python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3261,8 +3345,8 @@ def get_latest_materialization_code_versions(
 
         return result
 
-    @experimental
     @public
+    @experimental
```

Comment:
Minor consistency thing; feels like everywhere else in the codebase `@public` is above `@experimental`.